pom.xml

```diff
@@ -129,18 +130,35 @@
             <dependency>
                 <groupId>org.jenkins-ci</groupId>
                 <artifactId>symbol-annotation</artifactId>
-                <version>1.23</version>
+                <version>1.24</version>
             </dependency>
         </dependencies>
     </dependencyManagement>
 
     <!-- Pack into an HPI -->
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.13.0</version>
+                <configuration>
+                    <release>${java.level}</release>
+                </configuration>
+            </plugin>
             <plugin>
                 <groupId>org.jenkins-ci.tools</groupId>
                 <artifactId>maven-hpi-plugin</artifactId>
-                <version>3.19</version>
+                <configuration>
+                    <minimumJavaVersion>${java.level}</minimumJavaVersion>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.kohsuke</groupId>
+                <artifactId>access-modifier-checker</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
```

Comment:
Was this a required change as well?

Reply:
I don't know for sure if it's required but it doesn't build without it so I guess it's needed